zalo_sdk/oa/Client.py

```diff
@@ -63,6 +63,31 @@ def get_free_response_quota(self, message_id):
         response = self.send_request(
             method="POST", url=url, body=body, headers=headers)
         return self._validate_zalo_response(response)
+    
+    def get_zalo_oa_quotas(self, quota_owner: str, product_type: str = "", quota_type: str = ""):
```

Comment:
Check if all the input parameters are valid, e.g. `quota_owner` should be `OA` or `APP`; `product_type` and `quota_type` are optional but we should validate them if provided (see Zalo documents for the valid values)